lib/puppet/type/grafana_user.rb

```diff
@@ -57,7 +57,6 @@ def insync?(_is)
   newproperty(:is_admin) do
     desc 'Whether the user is a grafana admin'
     newvalues(:true, :false)
-    defaultto :false
```

Comment:
Now that `update_admin_flag` is only called if the property has been set, there is no need for a default here (and defaults for _properties_ are generally a bad thing).